kolibri/plugins/coach/assets/src/state/mutations/lessonsMutations.js

```diff
@@ -0,0 +1,36 @@
+export function SET_CLASS_LESSONS(state, lessons) {
+  state.pageState.lessons = [...lessons];
```

Comment:
is the goal of this pattern to make a copy?

Reply:
Yeah, but only shallow copies, so don't know how effective this is.

We do this because the Resource layer will sometimes modify data outside of a mutation, which can cause problems that are hard to trace.

Reply:
This PR is closed, David created a new one #3195

Reply:
whoops sorry, still had the tab open